simpleflow/execute.py

```diff
@@ -409,7 +411,25 @@ def main():
         metavar='N',
         help='error file descriptor',
     )
+    parser.add_argument(
+        '--kill-children',
+        action='store_const',
+        const=True,
+        default=False,
+        help='kill child processes on exit',
+    )
     cmd_arguments = parser.parse_args()
+
+    def kill_child_processes():
+        process = psutil.Process(os.getpid())
+        children = process.children(recursive=True)
+
+        for child in children:
```

Comment:
I think that even if they die by themselves it's ok since the references are kept (they are zombies).
```python
In [1]: import subprocess, psutil, os, time

In [2]: sleeper = subprocess.Popen(['sleep', '5']) ; children = psutil.Process(os.getpid()).children(recursive=True)

In [3]: time.sleep(6)

In [4]: children
Out[4]: [<psutil.Process(pid=15234, name='sleep') at 140707464425808>]

In [5]: [c.status() for c in children]
Out[5]: ['zombie']

In [6]: [c.terminate() for c in children]
Out[6]: [None]

In [7]: [c.status() for c in children]
Out[7]: ['zombie']

In [8]: [c.kill() for c in children]
Out[8]: [None]

In [9]: [c.status() for c in children]
Out[9]: ['zombie']
```